rebench/model/measurement.py

```diff
@@ -67,10 +67,16 @@ def line_number(self):
     TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
 
     def as_str_list(self):
-        return ["[" + self._timestamp.strftime(self.TIME_FORMAT) + "]",
```

Comment:
Stylistically, I would prefer if you handle the value just before the list creation.
So, something like `val = if then else; return [..., val, ...]` (that's just a sketch)